src/commonMain/kotlin/io/kotest/extensions/core/Validated.kt

```diff
@@ -1,4 +1,4 @@
-package io.kotest.assertions.arrow.core
+package io.kotest.extensions.core
```

Comment:
Do we want to change all the package names ?
These are assertions not extensions ?

Reply:
fixed :)